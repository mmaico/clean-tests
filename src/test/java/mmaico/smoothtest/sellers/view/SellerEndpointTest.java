package mmaico.smoothtest.sellers.view;



import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class SellerEndpointTest {


    @Test
    public void shouldRun() {

    }

}