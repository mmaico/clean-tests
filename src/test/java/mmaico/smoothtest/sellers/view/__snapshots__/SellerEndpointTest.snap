should_create_a_new_seller=[
  "{\"id\":\"4d48b4a9-890b-4b5d-b6cd-a4eaff0d8c11\",\"name\":\"Mandalorian\",\"enrollment\":\"2018-12-06T17:58:16.000+00:00\",\"level\":5,\"_links\":{\"self\":{\"href\":\"/sellers/4d48b4a9-890b-4b5d-b6cd-a4eaff0d8c11\"},\"hasContacts\":{\"href\":\"/sellers/4d48b4a9-890b-4b5d-b6cd-a4eaff0d8c11/contacts\"},\"hasScore\":{\"href\":\"/sellers/scores/5524\"}}}"
]


should_return_seller_by_id=[
  "{\"id\":\"53ac008b-9d16-4c36-afea-0e49c0c3515a\",\"name\":\"Boba Fett\",\"enrollment\":\"2018-12-06T17:58:16.000+00:00\",\"level\":5,\"_links\":{\"self\":{\"href\":\"/sellers/53ac008b-9d16-4c36-afea-0e49c0c3515a\"},\"hasContacts\":{\"href\":\"/sellers/53ac008b-9d16-4c36-afea-0e49c0c3515a/contacts\"},\"hasScore\":{\"href\":\"/sellers/scores/552544458\"}}}"
]